packages/core/src/nativeJs.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * @copyright (c) 2015-present, Philipp Thürwächter, Pattrick Hüper & js-joda contributors
+ * @license BSD-3-Clause (see LICENSE in the root directory of this source tree)
+ */
+
+import { requireNonNull } from './assert';
+import { IllegalArgumentException } from './errors';
+import { Instant, ZoneId } from './js-joda';
+
+/**
+ * Creates ZonedDateTime from a javascript Date or a moment instance.
+ * @param {!(Date|moment)} date - a javascript Date or a moment instance
+ * @param {ZoneId} [zone = ZoneId.systemDefault()] - the zone of the temporal, defaults to ZoneId.systemDefault()
+ * @returns {ZonedDateTime}
+ */
+export function nativeJs(date, zone = ZoneId.systemDefault()) {
+    requireNonNull(date, 'date');
+    requireNonNull(zone, 'zone');
```

Comment:
Do we need the check even though we have a default value?

Reply:
Of course we do! Default is for `undefined`. If you pass `null`, you get `null`.

Reply:
Hmm, if i want to break it, i can break it. I can't remember that we do null checks for defaults at other places.
Well but its fine for me, its kind of a double protection.

Reply:
You're right, you do not check for null for default at other places. I'll remove it then.

Reply:
... or I will not. You do check for null for default values sometimes (see https://github.com/js-joda/js-joda/blob/v2.2.1/packages/core/src/LocalDateTime.js#L289-L291). I think it is safer to keep null checks as a form of documentation.